release/release-trueos.sh

```diff
@@ -161,6 +161,23 @@ clean_poudriere()
 	echo -e "y\n" | poudriere ports -d -p ${POUDRIERE_PORTS}
 }
 
+super_clean_poudriere()
+{
+	#Look for any leftover mountpoints/directories and remove them too
+	for i in `ls ${POUDRIERED_DIR}/*-make.conf`
+	do
+		name=`basename "${i}" | rev | cut -d "-" -f 2-10 | rev`
```

Comment:
This looks *real* dangerous. If there are more or less '-' characters then something could go wrong here. 

Recommend you cd ${POUDRIERED_DIR} and ls '*-make.conf' instead. Then to get the jail/base name, you can do echo $i | sed 's|-make.conf||g' without that horrible rev / cut / rev stuff ;)

Reply:
Nevermind on the "cd" part, didn't see the 'basename' section.

Reply:
Done!